tests/shell/042_update_from_url/delete-files/input.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "type": "list",
+        "variable": "files",
+        "namespace": "delete_files",
+        "bundle": "delete_files",
+        "label": "Files",
+        "subtype": [
+        {
+            "key": "path",
+            "type": "string",
+            "label": "Path",
+            "question": "Path to file"
+        },
+        {
+            "key": "why",
+            "type": "string",
+            "label": "Why",
+            "question": "Why should this file be deleted?",
+            "default": "Unknown"
+        }
```

Comment:
Here and in cfbs.json you can maybe remove this object causing the input specification to change. This way we can pretend that this string field was added in the "newer" version of the module